pandas/tests/frame/methods/test_sort_values.py

```diff
@@ -497,3 +516,92 @@ def test_sort_values_ignore_index(
 
         tm.assert_frame_equal(result_df, expected)
         tm.assert_frame_equal(df, DataFrame(original_dict))
+
+
+class TestDataFrameSortKey:  # test key sorting (issue 27237), not yet implemented
+    @pytest.mark.skip(reason='key sorting is currently not implemented for dataframes')
```

Comment:
you can just xfail the entire class here; provide the message including the issue number IN the xfail reason).

do not skip tests.

Reply:
Some of the tests are fixtured. They succeed for some of the fixture parameters, and fail for others (they succeed for the None key but they fail for the identity key). I didn't see how I could resolve that.

Reply:
add strict = False

Reply:
I can just remove the fixture parameter in those functions for the time being if that's the easiest solution.

Reply:
done.